README.md

```diff
@@ -1,2 +1,4 @@
 # covid-19
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/db1476eb3cc745d8b2b82b762228c0b6)](https://app.codacy.com/gh/azaria-dedmon/covid-19?utm_source=github.com&utm_medium=referral&utm_content=azaria-dedmon/covid-19&utm_campaign=Badge_Grade_Settings)
\ No newline at end of file
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/db1476eb3cc745d8b2b82b762228c0b6)](https://app.codacy.com/gh/azaria-dedmon/covid-19?utm_source=github.com&utm_medium=referral&utm_content=azaria-dedmon/covid-19&utm_campaign=Badge_Grade_Settings)
+
+[![Coverage Status](https://coveralls.io/repos/github/azaria-dedmon/covid-19/badge.svg)](https://coveralls.io/github/azaria-dedmon/covid-19)
```

Comment:
@azaria-dedmon this is part of setting up the code coverage. Why is it in a different PR ? Please move these changes to the relevant PR and close this one